Makefile

```diff
@@ -25,6 +25,7 @@ checks:
 	./scripts/checks "$(PKGS)"
 
 build: deps checks
+	rm -rf Godeps/_workspace/pkg
```

Comment:
I guess I could call clean before build, but the important thing is that it happens before build.